tests/components/modbus/conftest.py

```diff
@@ -201,3 +201,21 @@ async def base_config_test(
         config_modbus=config_modbus,
         expect_init_to_fail=expect_init_to_fail,
     )
+
+
+async def prepare_service_update(hass, config, entity_id):
```

Comment:
Please remove the `entity_id` parameter. It's not used here.

Reply:
of course, done.